packages/excavator-projects/projects/dropBindlestocking.ts

```diff
@@ -0,0 +1,24 @@
+import { equippedAmount, Item } from "kolmafia";
+
+import { ExcavatorProject } from "../type";
+import { toNormalisedString } from "../utils";
+
+export const DROP_BINDLESTOCKING: ExcavatorProject = {
+  name: "Bindlestocking",
+  description: "Track critical drops from the bindlestocking.",
+  author: "Rinn",
+  hooks: {
+    COMBAT_ROUND: (_: string, page: string) => {
+      // Must be wielding a bindlestocking
+      if (equippedAmount(Item.get("bindlestocking")) < 1) return null;
+      const result = page.match(
+        /Something gets knocked loose from your bindlestocking!.*?You acquire an item: <b>(.*?)<\/b>/,
+      );
+      if (!result) return null;
+      const item = Item.get(result[1]);
```

Comment:
I think this needs to be `toItem` otherwise it throws an error if not found.

We should add a util that takes an item name and normalises it else returns the string, but don't need to do so now

Reply:
this is what we're currently doing for the other item drop equipment, so should prob handle all of it at once in a different PR